V2-launch-checklist.md

```diff
@@ -0,0 +1,155 @@
+- [ ] **TODO** Script to generate and upload all the above to IPFS
+  - Need to adapt: https://github.com/NexusMutual/smart-contracts/blob/782d93cf42dbc12d579b8625f3ec9b8d2c8c645f/scripts/populate-v2-products.js#L34
+- [ ] Upload the above to IPFS
+
+#3
+- [ ] **TODO** Format input for `setProducts()` and `setProductTypes()` functions 
+- [ ] **TODO**: Script to create the tx data for AB to write `products` and `productTypes` info 
+  onchain - we need to modify a bit `scripts/populate-v2-products.js` for this
+- [ ] Simulate the above txs in the fork test
+
+### Rewards
+**What we need**
+1. List of all V1 assessment rewards
+2. ... TBD
+
+**How we get it**
+- [ ] Run `get-legacy-assessment-rewards.js` to automatically populate `LegacyClaimsRewards.sol` with the list of addresses we reimburse with the assessment rewards.
+  - [ ] Check `LegacyClaimsRewards.sol`
+ 
+### Staking Pools - WIP
+- [ ] List of initial syndicate operators and their migration strategy
+  - [ ] Final inputs [here](https://docs.google.com/spreadsheets/d/1ebhsVWjc18rQJpGLMzRfmzRwwYzND7_6Q0A9zOlADvE)
+- [ ] Update `migrateToNewV2Pool` in `LegacyPooledStaking.sol` with the above info 
+
+### Onchain
+- [ ] **Add new proposal category**: 42 Add new contracts
+- [ ] **Add new proposal category**: 43 Remove contracts
+- [ ] **Edit proposal category** 41 (Set Asset Swap Details). It needs the new function signature in
+   order to modify limits used by `SwapOperator.sol`. 
+  - [ ] **TODO** double check if needed
+
+## Contract Deployment
+
+**Deployment & verification**
+Any time
+- [ ] `ProductsV1.sol` [non-internal]
+- [ ] `CoverNFT.sol` [non-internal] - *NFT symbols
+- [ ] `Assessment.sol` [AS] (must be done after `CoverNFT.sol`)
+- [ ] `IndividualClaims.sol` [IC] (must be done after `CoverNFT.sol`)
+- [ ] `YieldTokenIncidents.sol` [YT] (must be done after `CoverNFT.sol`)
+- [ ] `SwapOperator.sol` [non-internal]  
+- [ ] `PriceFeedOracle.sol` [non-internal]
+- [ ] `Pool.sol` [P1] (must be done after `SwapOperator.sol` and `PriceFeedOracle.sol`)
+  - [ ] !!! Contact Enzyme to whitelist the new Pool contract as receiver for the vault
+- [ ] `Governance.sol`[GV]
+- [ ] `CoverInitializer.sol` [CO]
+- [ ] `NXMaster.sol` [NXMaster]
+- [ ] `MCR.sol` [MC]
+- [ ] `MemberRoles.sol` [MR]
+- [ ] `Gateway.sol` [GW]
+- [ ] `CoverMigrator.sol`
+- [ ] `CoverViewer.sol`
+- [ ] `LegacyClaimsReward.sol` [CR]
+- [ ] `TokenController.sol` [TC] (must be done after `LegacyClaimsReward.sol`)
+
+After `CoverInitializer.sol` is upgraded(!)
+- [ ] `StakingPoolFactory.sol`
+- [ ] `StakingNFT.sol` - *NFT symbols
+- [ ] `StakingPool.sol` 
+- [ ] `LegacyPooledStaking.sol` [PS]
+- [ ] `Cover.sol` [CO] (must be done after `StakingNFT.sol`)
+
+## Upgrade
+
+### Pre-upgrade
+- [ ] Turn off the quote engine
```

Comment:
Before we turn it off we need to fetch all the prices.

Alternative: we only turn it off for outsiders, and run it in such a way only us can fetch latest prices.